includes/admin/tools/views/html-admin-page-exports.php

```diff
@@ -168,11 +168,27 @@ class="button-secondary"/>
 							      class="give-export-form">
 
 								<?php
-								$args = [
+								$args = array(
+									'id'          => 'give_donor_export_start_date',
+									'name'        => 'donor_export_start_date',
+									'placeholder' => esc_attr__( 'Start date', 'give' ),
+								);
+								echo Give()->html->date_field( $args ); ?>
+
+								<?php
```

Comment:
@mehul0810 All code should be in one PHP block `<?php ?>`, because their are not any HTML. You can use comment to differentiate